packages/blade/src/components/ActionList/ActionListItem.tsx

```diff
@@ -154,6 +164,30 @@ const ActionListItemIcon = assignWithoutSideEffects(_ActionListItemIcon, {
   componentId: componentIds.ActionListItemIcon,
 });
 
+const _ActionListItemBadgeGroup = ({
+  children,
+}: {
+  children: React.ReactElement[] | React.ReactElement;
+}): React.ReactElement => {
+  return (
+    <Box display="flex" alignItems="center" flexDirection="row">
+      {children}
+    </Box>
+  );
+};
+
+const ActionListItemBadgeGroup = assignWithoutSideEffects(_ActionListItemBadgeGroup, {
+  componentId: componentIds.ActionListItemBadgeGroup,
+});
+
+const _ActionListItemBadge = (props: BadgeProps): React.ReactElement => {
+  return <Badge size="medium" marginLeft="spacing.3" {...props} />;
```

Comment:
is badge size controlled by us or can be overriden by consumer? 

If we don't intend to let consumer override let's put size at the end.

Reply:
It will be consistent on design mostly. They can override though, its not breaking the UI so much. We just set the default size on our end.